### ISSUE TITLE
[Bug]: invalid content-type when generating a POST request with openapi generator

### ISSUE THREAD

**Issue by quentinderoubaix** [AmadeusITGroup/otter#788]
### Package name

generator-sdk

### Package version

Latest alpha

### Reproduction steps

Generate an api library using the newly schematic `ng generate sdk` and use a spec yaml file that contains a POST request with `consumes: -application/x-www-form-urlencoded`

Launch the typescript services / models generation.

### Current result

Observe that the typescript api will contain the following:
```typescript
  const headers: { [key: string]: string | undefined } = { 
    'Content-Type': metadata?.headerContentType || 'application/json',
    ...(metadataHeaderAccept ? {'Accept': metadataHeaderAccept} : {})
  };
```

### Expected result

The part above should be:
```typescript
  const headers: { [key: string]: string | undefined } = { 
    'Content-Type': metadata?.headerContentType || 'application/x-www-form-urlencoded',
    ...(metadataHeaderAccept ? {'Accept': metadataHeaderAccept} : {})
  };
```

### Additional comments

This is due to a regression in the api.mustache file when switching to the openapi generator.
We are using HeaderJsonMimeType custom lambda but giving an invalid input.

**Comment by kpanot** [comment]
The internal SDK may need to be re-generated with the latest version of the generator including the support of the `consumes` headers.
- [ ] regeneration of `@dapi/amadeus-gateway-sdk` based Otter 9.1+

**Comment by mrednic-1A** [comment]
Thanks @quentinderoubaix for your contribution.
We have included your fix in the version 9.1 of the  ama-sdk-schematics (sdk generator) package.
